Matlab/Miscellaneous.tmPreferences

```diff
@@ -34,21 +34,24 @@
 		<array>
 			<dict>
 				<key>name</key>
-				<string>TM_COMMENT_START</string>
+				<string>TM_COMMENT_START_2</string>
```

Comment:
What does mean `proper commenting functionality, i.e., override the TM_COMMENT_* variables for source.octave source.matlab`?

`crippled by this`, it is a number, what matter if it is 0 or 2? It still working the same way for matlab. Instead of cripple, I am letting the default Matlab syntax to be openly compatible for extension. If the number can start on 2 without any prejudice for anything, except now you allow it to be extensible for Octave as below:

Do you know how to fix this without changing the Matlab initial sequence to 2? I tried everything, rename the files, variables, restart several times Sublime Text, reload the syntax, several times, everything is useless. Do you really think opening it to extension is to cripple? I would agree if it would interfere with the current Matlab syntax, but it is does nothing. Why do you insist so much in start the number on 0? 

1. Starting it on 2 allow extension, 0 does not.
1. Starting on 2 does not affect the Matlab syntax
1. And starting it on 0 does not allow Octave's extension.

Are you saying the default syntaxes are forbidden to be adapt to extension at the cost of absolutely nothing? This would be just insane.

If there is a cost for the Matlab syntax, would be perfectly understandable to not change the start number from 0 to 2.

There is nothing else I can find out to do on the Octave's file to fix it. The octave syntax is:
```
%YAML 1.2
---
name: OCTAVE
file_extensions:
  - octave
scope: source.octave
contexts:
  main:
    - match: ''
      push: Packages/Matlab/Matlab.sublime-syntax
      with_prototype:
        - match: '^\s*[#%]{\s*\n'
          scope: comment.line.octave
          push:
            - meta_scope: comment.line.octave
            - match: '^\s*[#%]}\s*\n'
              scope: comment.line.octave
              pop: true
        - match: '#.*\n'
          scope: comment.line.percentage.octave
```
Octave/Miscellaneous.tmPreferences
```
...
		<array>
			<dict>
				<key>name</key>
				<string>TM_COMMENT_START</string>
				<key>value</key>
				<string># </string>
			</dict>
			<dict>
				<key>name</key>
				<string>TM_COMMENT_START_2</string>
				<key>value</key>
				<string>#{
</string>
			</dict>
			<dict>
				<key>name</key>
				<string>TM_COMMENT_END_2</string>
				<key>value</key>
				<string>
#}
</string>
			</dict>
		</array>
...
```

Matlab/Miscellaneous.tmPreferences
```
...
		<array>
			<dict>
				<key>name</key>
				<string>TM_COMMENT_START_2</string>
				<key>value</key>
				<string>% </string>
			</dict>
			<dict>
				<key>name</key>
				<string>TM_COMMENT_START_3</string>
				<key>value</key>
				<string>%{
</string>
			</dict>
			<dict>
				<key>name</key>
				<string>TM_COMMENT_END_3</string>
				<key>value</key>
				<string>
%}
</string>
			</dict>
		</array>
...
```

I am properly setting on each file:
```
	<key>name</key>
	<string>Miscellaneous</string>
	<key>scope</key>
	<string>source.octave</string>
...
	<key>name</key>
	<string>Miscellaneous</string>
	<key>scope</key>
	<string>source.matlab</string>
```